Generating bindings for ESP-IDF 5.0 / master (2022-05-26) fails on fatal error: 'esp_int_wdt.h' file not found

Scenario:
- open a project with activated ESP-IDF from master (5.0)
- apply workaround for #99 
- build a project
- build fails:
```
  [815/815] cd /workspace/rustzx-esp32/target/xtensa-esp32-espidf/release/build/esp-idf-sys-7fb14e0cab432a5e/out/build/esp-idf/esptool_py && python /home/gitpod/.espressif/frameworks/esp-idf-master/components/partition_table/check_sizes.py --offset 0x8000 partition --type app /workspace/rustzx-esp32/target/xtensa-esp32-espidf/release/build/esp-idf-sys-7fb14e0cab432a5e/out/build/partition_table/partition-table.bin /workspace/rustzx-esp32/target/xtensa-esp32-espidf/release/build/esp-idf-sys-7fb14e0cab432a5e/out/build/libespidf.bin
  libespidf.bin binary size 0x27440 bytes. Smallest app partition is 0x100000 bytes. 0xd8bc0 bytes (85%) free.
  cargo:root=/workspace/rustzx-esp32/target/xtensa-esp32-espidf/release/build/esp-idf-sys-7fb14e0cab432a5e/out
  cargo:rerun-if-changed=/home/gitpod/.cargo/registry/src/github.com-1ecc6299db9ec823/esp-idf-sys-0.31.5/src/include/esp-idf/bindings.h
...
 /home/gitpod/.cargo/registry/src/github.com-1ecc6299db9ec823/esp-idf-sys-0.31.5/src/include/esp-idf/bindings.h:14:10: fatal error: 'esp_int_wdt.h' file not found
  /home/gitpod/.cargo/registry/src/github.com-1ecc6299db9ec823/esp-idf-sys-0.31.5/src/include/esp-idf/bindings.h:14:10: fatal error: 'esp_int_wdt.h' file not found, err: true
```

How to simulate:

- open: https://gitpod.io/github.com/georgik/rustzx-esp32/tree/feature/feature/esp32-esp-idf-5.0
- run: 
```
mkdir .embuild/espressif/python_env # Workaround of 99
ln -s ~/.espressif/python_env/idf5.0_py3.8_env/ .embuild/espressif/python_env/idf5.0_py3.8_env
./run-wokwi.sh
```

--- a/src/include/esp-idf/bindings.h
+++ b/src/include/esp-idf/bindings.h
@@ -11,9 +11,14 @@
 #include "esp_sleep.h"
 #include "esp_task.h"
 #include "esp_task_wdt.h"
-#include "esp_int_wdt.h"
 #include "esp_interface.h"
 #include "esp_ipc.h"
+
+#if ESP_IDF_VERSION_MAJOR == 4
+#include "esp_int_wdt.h"
+#elif ESP_IDF_VERSION_MAJOR == 5
+#include "esp_private/esp_int_wdt.h"
+#endif
 
 #ifdef ESP_IDF_COMP_CONSOLE_ENABLED
 #include "esp_console.h"

